compiler/env/OMRCPU.cpp

```diff
@@ -29,15 +29,32 @@
 #include "env/defines.h"
 #include "omrcfg.h"
 
+namespace OMR
+{
```

Comment:
I thought the convention in the compiler was to not wrap function definitions in a namespace declaration and use namespace prefixes (i.e. `OMR::`) instead?